server/src/server.ts

```diff
@@ -41,6 +42,8 @@ let projectsFiles: Map<
   {
     openFiles: Set<string>;
     filesWithDiagnostics: Set<string>;
+    filesDiagnostics: filesDiagnostics;
```

Comment:
These might be possible to unify at some point, but I want to make minimal changes in this PR.